### PR TITLE
new mesa and xorg binaries

### PR DESCRIPTION
**Pull request by satmandu**
-mesa -> 21.0.0
-xorg_server  add binaries

Works properly:
- [x] x86_64

Builds properly:
- [x] x86_64
- [x] i686
- [x] armv7l

### CODE REVIEW

File: packages/xorg_server.rb [hunk 1]
```diff
@@ -41,12 +50,12 @@ class Xorg_server < Package
...
-    when 'i686'
-      PEER_CMD_PREFIX='/lib/ld-linux-i686.so.2'
-    when 'x86_64'
-      PEER_CMD_PREFIX='/lib64/ld-linux-x86-64.so.2'
+  when 'armv7l', 'aarch64'
+    PEER_CMD_PREFIX = '/lib/ld-linux-armhf.so.3'.freeze
+  when 'i686'
+    PEER_CMD_PREFIX = '/lib/ld-linux-i686.so.2'.freeze
+  when 'x86_64'
+    PEER_CMD_PREFIX = '/lib64/ld-linux-x86-64.so.2'.freeze
```
Comment (uberhacker) [inline packages/xorg_server.rb:58]: Why the `.freeze`?

Comment (satmandu) [reply packages/xorg_server.rb:58]: rubocop... is it bad?

Comment (uberhacker) [reply packages/xorg_server.rb:58]: Just not understanding what needs to be "frozen".

Comment (satmandu) [reply packages/xorg_server.rb:58]:
```
Offenses:

forupload/xorg_server.rb:54:23: C: [Corrected] Style/MutableConstant: Freeze mutable objects assigned to constants.
    PEER_CMD_PREFIX = '/lib/ld-linux-armhf.so.3'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^
forupload/xorg_server.rb:57:23: C: [Corrected] Style/MutableConstant: Freeze mutable objects assigned to constants.
    PEER_CMD_PREFIX = '/lib/ld-linux-i686.so.2'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^
forupload/xorg_server.rb:60:23: C: [Corrected] Style/MutableConstant: Freeze mutable objects assigned to constants.
    PEER_CMD_PREFIX = '/lib64/ld-linux-x86-64.so.2'
                      ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

let me change those constants..

Comment (satmandu) [reply packages/xorg_server.rb]: There...

**Pull request merged**